src/main/java/com/microsoft/graph/tasks/LargeFileUploadTask.java

```diff
@@ -292,17 +292,17 @@ private IUploadSession extractSessionFromParsable(@Nonnull Parsable uploadSessio
         session.setNextExpectedRanges((List<String>) uploadSession.getClass().getDeclaredMethod("getNextExpectedRanges").invoke(uploadSession));
         return session;
     }
-    private long nextSliceSize(long rangeBegin, long rangeEnd) {
+    private long nextSliceSize(long rangeBegin, long rangeEnd)  {
         long size = rangeEnd - rangeBegin + 1;
         return Math.min(size, this.maxSliceSize);
     }
-    private byte[] chunkInputStream(InputStream stream, int begin, int length) {
+    private byte[] chunkInputStream(InputStream stream, int begin, int length) throws Throwable {
```

Comment:
arguable we should declare throwing an IOException instead so people can write code specifically catching that. 
It's a bit leaking the implementation into the contract so what I usually do in those cases is wrap the specific exception into a runtime exception.